src/interface/meta.thrift

```diff
@@ -124,16 +123,10 @@ struct IndexFields {
     1: map<string, list<common.ColumnDef>>(cpp.template = "std::map")  fields,
 }
 
-struct TagIndexItem {
-    1: common.TagIndexID    index_id,
+struct IndexItem {
+    1: common.IndexID       index_id,
     2: string               index_name,
-    3: IndexFields          fields,
-}
-
-struct EdgeIndexItem {
-    1: common.EdgeIndexID   index_id,
-    2: string               index_name,
-    3: IndexFields          fields ,
+    4: IndexFields          fields,
 }
```

Comment:
Could we move the `IndexItem` to `common.thrift`? Because `IndexItem` is used by storage layer and graph layer.😊

Reply:
Good Idea!